### DISCUSSION TITLE
Every interest record is a record in market data?

### DISCUSSION THREAD

**Discussion by morskipas** [ghostfolio/ghostfolio#3890] [Q&A]
Hello everyone,
I have set up a money market account and am now tracking interest payments. After entering the interest as an activity, it is displayed correctly. However, I have also noticed that every single interest payment is also displayed under market data. 
Is this intended?

Best regards,
Frank

![image](https://github.com/user-attachments/assets/4286825f-0c53-47c6-badc-a6e9c9e773d9)

**Comment by dtslvr** [comment]
Hi @morskipas

This topic is a duplicate of https://github.com/ghostfolio/ghostfolio/issues/3381. We're closing this one to consolidate discussion. Please follow the progress and updates on the original issue.
